test-toolkit/docker/1.6.0/Dockerfile.mxnet.cpu

```diff
@@ -0,0 +1,68 @@
+FROM ubuntu:18.04
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+ARG MMS_VERSION=1.0.8
+
+ENV SAGEMAKER_SERVING_MODULE sagemaker_mxnet_serving_container.serving:main
+ENV TEMP=/home/model-server/tmp
+
+# No interactive mode to install OpenCV
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    software-properties-common \
+    build-essential \
+    ca-certificates \
+    curl \
+    git \
+    openssh-client \
+    openssh-server \
+    vim \
+    wget \
+    python3-dev \
+    && ln -s -f /usr/bin/python3 /usr/bin/python \
+    && ln -s /usr/local/bin/pip3 /usr/bin/pip
+
+RUN apt-get install -y --no-install-recommends \
+    libgl1-mesa-glx \
+    libglib2.0-0 \
+    libopencv-dev \
+    libsm6 \
+    libxext6 \
+    libxrender-dev \
+    openjdk-8-jdk-headless \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# MXNet requires pip 19.3.1 due to being backwards compatible
```

Comment:
nit: do you mean "due to being backwards incompatible" or "so that it's backwards compatible" instead of "due to being backwards compatible"?

Reply:
copied from the DLC dockerfile